1800clothing/carma/templates/scss/styles.css

```diff
@@ -688,6 +680,43 @@ footer p {
 header {
   top: 0; }
 
+.downloadsBody {
+  text-align: center;
+  padding-top: 58px; }
+  .downloadsBody h1 {
+    font-weight: bold;
+    margin: 0;
+    padding: 30px;
+    font-size: 50px; }
+  .downloadsBody p {
+    margin: 0;
+    padding: 30px;
+    padding-top: 0;
+    font-size: 30px; }
+  .downloadsBody .downloadLinks {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-gap: 20px;
+    max-width: 500px;
+    margin: 0 auto; }
+    .downloadsBody .downloadLinks a {
+      text-decoration: none; }
+      .downloadsBody .downloadLinks a .downloadBtn {
+        color: white;
+        background-color: #2E2260;
+        padding: 15px;
+        font-size: 28px;
+        border-radius: 10px;
+        box-shadow: 4px 4px 13px 3px rgba(0, 0, 0, 0.13);
+        transition: 0.3s; }
+      .downloadsBody .downloadLinks a .downloadBtn:hover {
+        background-color: #453391; }
+
+@media only screen and (max-width: 593px) {
+  .downloadsBody .downloadLinks {
+    grid-template-columns: 1fr; }
+    .downloadsBody .downloadLinks a {
+      margin: 0 30px; } }
 @media only screen and (min-width: 30rem) {
   #aside1 {
     grid-area: aside1; }
```

Comment:
Your css is fine but the line formatting isn't standard practice. The closing bracket should be on a new line, for readability.